### ISSUE TITLE
So far not working well

### ISSUE THREAD

**Issue by amd64fox** [kyrie25/Spicetify-Cache-Cleaner#1]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.92.647.ga4397eb7
- Spicetify version: dev
```

### 📝 Description

Doesn't clear cache after startup

### 📸 Screenshots
**_Notification after launch_**
![Снимок экрана 2022-08-18 195030](https://user-images.githubusercontent.com/62529699/185452043-f4a71a5f-c2f9-4dd0-a8c0-bd09007e6b86.jpg)

**_Settings_**
![Снимок экрана 2022-08-18 195053](https://user-images.githubusercontent.com/62529699/185452059-c303f449-370a-4d0c-b9ee-652bf81c7eff.jpg)

**Comment by kyrie25** [comment]
Does it work as expected when it is manually cleared?

**Comment by kyrie25** [comment]
What makes this interesting is that the notification is only called *after* the promise has been fulfilled. Seems that it is dependent on another Spotify service on startup also.
https://github.com/kyrie25/Spicetify-Cache-Cleaner/blob/9af224fde07bbfb9f1c0a572a2208016bd0b481e/cacheCleaner.js#L38-L59

**Comment by kyrie25** [comment]
Resolved via 2c81de1
